spec/rubocop/cop/rails/redundant_active_record_all_method_spec.rb

```diff
@@ -260,6 +260,26 @@
         RUBY
       end
     end
+
+    context 'when `all` has any parameters (indicating it is not an Active Record `all`)' do
```

Comment:
Can you tweak the text?

```suggestion
    context 'when `all` has any parameters, it indicates that it is not an Active Record `all`' do
```

Reply:
Thank you for the review. I've updated the context name.